build-configs/common/theme/colors.ts

```diff
@@ -19,6 +20,7 @@ export const commonLightColors = {
   textSecondaryColor: '#585858',
   textDecorationColor: '#c7c7c7',
   textDisabledColor: '#d0d0d0',
+  footerLineColor: 'b1b1b1',
```

Comment:
Even so it works fine, I would prefer the hash-prefixed variant for consistency:
```suggestion
  footerLineColor: '#b1b1b1',
```